engine/Engine_Phonotype.sc

```diff
@@ -32,6 +32,17 @@ Engine_Phonotype : CroneEngine {
 			};
 
 		};
+
+		// does this auto wire things together? where should the OSCdef live?
```

Comment:
It's okay for the OscDef to live here, but make an `oscCrowOut` instance variable in the class and create it with 

```
oscCrowOut = OSCdef.new(...)
```

... then free it in the cleanup method.